ServerlessLibraryAPI/ClientApp/src/components/DetailView/DetailPageContent.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from "react";
+import ReactMarkdown from "markdown-to-jsx";
+import {
+  Pivot,
+  PivotItem,
+  PivotLinkSize,
+  ScrollablePane
+} from "office-ui-fabric-react/lib/index";
+import { getTheme, mergeStyleSets } from "office-ui-fabric-react";
+import "./DetailPageContent.scss";
+const theme = getTheme();
+
+const classNames = mergeStyleSets({
+  wrapper: {
+    minHeight: "60vh",
+    position: "relative",
+    maxHeight: "inherit"
+  },
+  pane: {
+    minHeight: "60vh",
+    border: "0px solid " + theme.palette.neutralLight,
+    whiteSpace: "pre",
+    backgroundColor: "#FBFBFB",
+    width: "80%"
+  },
+  textContent: {
+    padding: "15px 10px"
+  }
+});
+
+class DetailPageContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      armTemplateText: "",
+      markdownText: "",
+      selectedKey: "licence"
+    };
+
+    this.handleLinkClick = this.handleLinkClick.bind(this);
+  }
+
+  handleLinkClick(pivotItem, ev) {
+    this.setState({
+      selectedKey: pivotItem.props.itemKey
+    });
+    if (
+      pivotItem.props.itemKey === "armtemplate" &&
+      this.state.armTemplateText === ""
+    ) {
+      let { template } = this.props;
+      if (template) {
+        fetch(template)
+          .then(response => response.text())
```

Comment:
you can add handling for text responses in handle-response.js -> we can build that helper to handle negative scenarios properly in a single place

Reply:
will address this later once we freeze on the behaviour for error handling